packages/ui-tabs/src/Tabs/styles.ts

```diff
@@ -131,6 +131,21 @@ const generateStyle = (
         background: `linear-gradient(to right, ${componentTheme.scrollFadeColor} 0%, rgba(255, 255, 255, 0) 100%)`
       }
     },
+    leftScrollOverlay: {
+      label: 'tabs__scrollOverlay',
+      height: `calc(100% - (${componentTheme.tabVerticalOffset} + 0.25rem))`,
```

Comment:
It is not necessary, but I recommend refactoring the common overlay styles into a const (that are the same for both the left and right overlay), because then you don't have to update it in 2 places later.